Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/ForwardPass.azsli

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+// These setting used to be in ShaderQualityOptions.azsli as a basic/limited solution for multiple pipelines.
+// TODO, instead of UNIFIED_FORWARD_OUTPUT, fork ForwardPassOutput.azsli
+#define UNIFIED_FORWARD_OUTPUT          1
+#define FORCE_IBL_IN_FORWARD_PASS       1
+#define ENABLE_AREA_LIGHT_VALIDATION    0
+// #define ENABLE_AREA_LIGHTS           0
+
+#include <viewsrg.srgi>
+#include <Atom/Features/PBR/DefaultObjectSrg.azsli>
+#include <Atom/Features/Pipeline/Forward/ForwardPassSrg.azsli>
+#include <Atom/Features/Pipeline/Forward/ForwardPassOutput.azsli>
+#include <Atom/Features/PBR/AlphaUtils.azsli>
+#include <Atom/Features/SrgSemantics.azsli>
+#include <Atom/Features/ColorManagement/TransformColor.azsli>
+#include <Atom/Features/PBR/Lighting/StandardLighting.azsli>
+#include <Atom/Features/PBR/Decals.azsli>
+
+struct VSInput
+{
+    float3 m_position : POSITION;
+    float3 m_normal : NORMAL;
+    float4 m_tangent : TANGENT; 
+    float3 m_bitangent : BITANGENT; 
+};
+
+struct VSOutput
+{
+    precise linear centroid float4 m_position : SV_Position;
+    float3 m_normal: NORMAL;
+    float3 m_tangent : TANGENT; 
+    float3 m_bitangent : BITANGENT; 
+    float3 m_worldPosition : UV0;
+};
+
+#include <Atom/Features/Vertex/VertexHelper.azsli>
```

Comment:
VertexHelper is removed since it only does world position and TBN calculation, the latter being moved to pixel shader.
Expanding VertexHelper(IN, OUT, worldPosition) should cause less trouble when merging development.